sefaria/helper/crm/salesforce.py

```diff
@@ -6,6 +6,9 @@
 from sefaria.helper.crm.crm_connection_manager import CrmConnectionManager
 from sefaria import settings as sls
 
+from typing import Any, Optional, List
```

Comment:
Use of typing is highly appreciated! Note, `List` is no longer necessary to import in Python 3.9 which we now use on prod. use the built-in `list` type instead.